src/system/AdminModule/Controller/AdminInterfaceController.php

```diff
@@ -0,0 +1,579 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2009 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\AdminModule\Controller;
+
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Component\HttpFoundation\Response;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; // used in annotations - do not remove
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method; // used in annotations - do not remove
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Zikula\Core\Controller\AbstractController;
+use Zikula\Bundle\CoreBundle\Console\Application;
+use Zikula\Core\Theme\Annotation\Theme;
+
+
+/**
+ * @Route("/admininterface")
+ */
+class AdminInterfaceController extends AbstractController
+{	
+	/**
+	 * @Route("/header")
+	 *
+	 * @Template
```

Comment:
I just read here http://symfony.com/doc/current/best_practices/controllers.html#template-configuration that use of `@Template` is not within Symfony's "Best Practices" design. @Guite @cmfcmf thoughts about whether the Core should utilize such annotations? (if I not I also have some refactoring to do...)